dashboard/app/models/ability.rb

```diff
@@ -124,6 +125,15 @@ def initialize(user)
       can :project_commits, ProjectVersion do |_, project_owner, project_id|
         CodeReviewComment.user_can_review_project?(project_owner, user, project_id)
       end
+
+      can :create, CodeReview do |code_review, project_owner_id|
+        code_review.user_id == user.id &&
+        project_owner_id == user.id
+      end
+      can :edit, CodeReview, user_id: user.id
+      # TODO: teachers and peers should also be able to see the code review
```

Comment:
Do you want to create a ticket for this? Or just planning on addressing it with other work?

Reply:
This PR was kind of a checkpoint and I plan on addressing this in the next PR.  If it goes beyond that, I will open a ticket to track.